pathmux/viztree.go

```diff
@@ -0,0 +1,148 @@
+package pathmux
+
+// Exploded version of the pathmux tree designed for being easy to use in a visualization.
+// Simple wildcard nodes are represented by the ':' prefix and free wildcard nodes with the '*' prefix.
+type VizTree struct {
+	Path     string // string representation of the node path
+	children []*VizTree
```

Comment:
wondering if it would be more useful to make the `children` field visible to the calling code. It would make the viztree walkable. Is there a reason not to do it?